docs/NRI.md

```diff
@@ -108,12 +108,11 @@ For more details about pre-registered and externally launched plugins
 
 You can verify that NRI integration is properly enabled and functional by
 configuring containerd and NRI as described above, taking the NRI
-logger plugin either from the NRI repository or
-[this fork](https://github.com/klihub/nri/tree/pr/proto/draft/plugins/logger)
+logger plugin either from the [NRI repository](https://github.com/containerd/nri/tree/main/plugins/logger)
```

Comment:
minor word issue: since there aren't two options now the word "either" needs to be removed since you just get the plugin "from the [NRI repository]...." now.